### PR TITLE
tutorial.md updated

### PR DESCRIPTION
**Pull request by jesusgonzalez12**
Updated the file with some syntax fixes and now it also is standard compliant.

### CODE REVIEW

File: docs/collections/documents/geospatial/tutorial.md [hunk 1]
```diff
@@ -212,136 +212,160 @@ client.delete_collection(collection_shops)
...
+const globalUrl = "https://gdn.paas.macrometa.io";
+
+// Create a authenticated instance with Token / Apikey
+// const client = new jsc8({ url: globalUrl, token: "XXXX", fabricName: '_system' });
+// Step 1: Open connection to GDN. You will be routed to the closest region.
+console.log(`\n1. Connect to federation: ${globalUrl}`);
+const client = new jsc8({ url: globalUrl, apiKey: "XXXXX", fabricName: "_system" });
+console.log("Connected: Authentication done!!...");
+
+// Or use Email & Password to Authenticate client instance
```
Comment (diana-macrometa) [inline docs/collections/documents/geospatial/tutorial.md]:
```suggestion
// Or use email and password to authenticate client instance
```

File: docs/collections/documents/geospatial/tutorial.md [hunk 2]
```diff
@@ -212,136 +212,160 @@ client.delete_collection(collection_shops)
...
-        if (exists_transit === false) {
-              await client.createCollection(collection_transit)
-              await client.addGeoIndex(collection_transit, ['coordinate'],{geoJson : false});
-        }
+const jsc8 = require("jsc8");
+
+// Variables - DB
+const globalUrl = "https://gdn.paas.macrometa.io";
+
+// Create a authenticated instance with Token / Apikey
```
Comment (diana-macrometa) [inline docs/collections/documents/geospatial/tutorial.md]:
```suggestion
// Create a authenticated instance with token or API key
```

**Review by diana-macrometa — APPROVED** [review]

**Review by diana-macrometa — APPROVED** [review]

**Comment by jesusgonzalez12** [comment]
Hi @diana-macrometa. Thanks for the feedback.
We could wait for tomorrow for the other reviewers to see their comments.
Other than that, we're ready for a merge here.

File: docs/collections/documents/geospatial/tutorial.md [hunk 3]
```diff
@@ -212,136 +212,160 @@ client.delete_collection(collection_shops)
...
-        if (exists_transit === false) {
-              await client.createCollection(collection_transit)
-              await client.addGeoIndex(collection_transit, ['coordinate'],{geoJson : false});
-        }
+const jsc8 = require("jsc8");
+
+// Variables - DB
+const globalUrl = "https://gdn.paas.macrometa.io";
+
+// Create a authenticated instance with token or API key
```
Comment (diana-macrometa) [inline docs/collections/documents/geospatial/tutorial.md]:
```suggestion
// Create an authenticated instance with token or API key
```

File: docs/collections/documents/geospatial/tutorial.md [hunk 4]
```diff
@@ -212,136 +212,160 @@ client.delete_collection(collection_shops)
...
+  const lat2 = 37.332401582858324;
+  const long2 = -121.80235913612003;
+
+  console.log(`\n4c. Find buses available within rectangular fence: (${lat1}, ${long1}, ${lat2}, ${long2})`);
+
+  withinQuery = `FOR loc IN WITHIN_RECTANGLE (${collectionTransit}, ${lat1}, ${long1}, ${lat2}, ${long2}) RETURN loc`;
+  result = await client.executeQuery(withinQuery);
+  console.log(result);
 
   // Find Bus closest to given location
```
Comment (diana-macrometa) [inline docs/collections/documents/geospatial/tutorial.md]:
```suggestion
  // Find bus closest to given location
```

**Review by diana-macrometa — APPROVED** [review]
Approved with minor edits

**Pull request merged**